Decred Wallet/en.lproj/Localizable.strings

```diff
@@ -383,3 +383,14 @@
 "errorMsg" = "Error Message";
 "info" = "Info";
 "delete" = "Delete";
+"allowOnce" = "Allow once";
+"alwaysAllow" = "Always allow";
+"notNow" = "Not now";
+
+/* Other Text */
+"send" = "send";
+"sent" = "sent";
+"receive" = "receive";
+"history" = "history";
+"settings" = "Settings";
+"help" = "help";
```

Comment:
This translation file has 396 lines, the others have 400.

Reply:
resolved in 2c74553

Reply:
this was due to changes in the navigation menu items. resolved by making all variables read the same in their localized strings file in 2c74553